test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -127,9 +127,9 @@ private IEnumerable<string> GetConfigurationDirs()
 
             if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
             {
-                if(RuntimeInformation.ProcessArchitecture == Architecture.X64)
+                if (RuntimeInformation.ProcessArchitecture == Architecture.X64)
```

Comment:
Sounds good. We updated the .editorconfig file to match with Roslyn, and this is one of the changes it wanted. To prevent a PR that had every file in the repo changed, we have been doing the PR's one project at a time. I haven't don't this project yet, so the files I opened in Visual Studio wanted the fixes done. I'll revert just the formatting changes, and then put the whole project formatting changes as a new PR.